src/config/default-config.js

```diff
@@ -50,7 +50,7 @@ async function getAwsPoolId() {
   // when k8s is undefined we are in development where we use staging user pool so we set
   // it as the default one.
   const k8sEnv = process.env.K8S_ENV || 'staging';
-  const userPoolName = `biomage-user-pool-case-insensitive-${k8sEnv}`;
+  const userPoolName = `test-biomage-user-pool-${k8sEnv}`;
 
   const poolId = UserPools.find((pool) => pool.Name === userPoolName).Id;
   return poolId;
```

Comment:
All of this is temporary changes so that we can test the changes done in a custom userPool instead of the real staigng/prod ones